app/presenters/grove_monitor_presenter.rb

```diff
@@ -2,10 +2,16 @@ class GroveMonitorPresenter
   attr_reader :lost_students, :absent_students, :locations
   def initialize(grove)
     @absent_students = grove.students.absent
-    @lost_students = grove.students.lost
+    @lost_students = grove.students.lost.where(at_school: true)
+
```

Comment:
You can actually just get rid of the `.where(at_school: true)` I added it to the Student model in my PR. Also that was my bad for not putting it there to begin with.